stock_immediate_transfer_disable/wizard/stock_immediate_transfer.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <record id="view_immediate_transfer" model="ir.ui.view">
+        <field name="name">stock.immediate.transfer.view.form</field>
+        <field name="model">stock.immediate.transfer</field>
+        <field name="inherit_id" ref="stock.view_immediate_transfer"/>
+        <field name="arch" type="xml">
+            <form position="replace">
+                <form string="No Quantities Selected">
+                    <field name="allow_imediate_transfer" invisible="1"/>
+                    <group attrs="{'invisible': [('allow_imediate_transfer', '=', False)]}">
```

Comment:
@dufresnedavid Maybe you could inherit the view here in a way to avoid doubling the text that's already present in the parent view.

Reply:
@joaoalf I though about that, but this would not make the code more maintainable.

The original form hardly inheritable. I would need otherwise to insert multiple xpaths. I prefer keeping it simple if you don't mind.